flag:
  show_time: false
  show_image: true
  use_nuscenes: true

# 0 - vinilla, 1 - yolov5, 2 - euclideancluster, 3 - virtual point
mode: 0

camera:
  topic: /cam_front/raw
  frame_id: cam_front
  image_width: 1600
  image_height: 900
  intrinsic:
    [
      1266.417203046554,
      0.0,
      816.2670197447984,
      0.0,
      1266.417203046554,
      491.50706579294757,
      0.0,
      0.0,
      1.0,
    ]
  projection:
    [
      1266.417203046554,
      0.0,
      816.2670197447984,
      0.000000,
      0.000000,
      1266.417203046554,
      491.50706579294757,
      0.000000,
      0.000000,
      0.000000,
      1.000000,
      0.000000,
    ]
  distortion: [0.000000, 0.000000, 0.000000, 0.000000, 0.000000]

lidar:
  topic: /lidar_top
  frame_id: lidar_top
  rings: 32
  # frame lidar right - x+, front - y+, up - z+
  # frame camera right - x+, down - y+, front - z+
  filter:
    xmin: -100.0
    xmax: 100.0
    ymin: -100.0
    ymax: 100.0
    zmin: 0.0
    zmax: 100.0
  ec_extraction:
    cluster_tolerance: 0.1
    min_cluster_size: 20
    max_cluster_size: 25000
  max_distance: 100.0

transform:
  lidar2camera:
    [
      0.0645287,
      -0.997914,
      0.00164512,
      -0.0251032,
      -0.0351988,
      -0.0039236,
      -0.999373,
      0.0504193,
      0.997295,
      0.0644304,
      -0.0353786,
      -0.205162,
      0.000000,
      0.000000,
      0.000000,
      1.000000,
    ]
